site-admin.md

```diff
@@ -0,0 +1,499 @@
+manually.
+
+Important: whenever the website's dependencies change or are updated, the lock
+file `package-lock.json` will be updated. Whenever `package-lock.json` is
+updated, you should re-run `npm install` (on CI, `npm ci`), so that your local
+environment's dependencies will match the production environment.
+
+## Orientation 🗺
+
+Here is some basic info to help orient you to this repo.
+
+- This repo holds the **code** for two websites:
+  - `./apps/labs/` holds code for Quansight Labs: <https://labs.quansight.org>
+  - `./apps/consulting/` holds code for the previous Quansight Consulting website.
+    It is now just an archive.
+  - `./libs` holds code shared by both websites.
+- Labs website **content** lives in [Storyblok](https://app.storyblok.com)
+  (requires login).
+  - But Labs **blog posts** live under `./apps/labs/posts`
+- The websites are hosted and deployed via
+  [Vercel](https://vercel.com/quansight) (requires login).
+- The repo's default branch is `main`
+  - Most pull requests (i.e., Labs blog posts) will be opened against
+    `main`.
+  - Riskier pull requests will be opened against the `develop` branch.
+  - `main` is used for production (i.e., the live website).
+  - You can think of `develop` as staging.
+  - Pushing commits to `main` triggers a deployment of both websites via Vercel.
+
+## Deployment Schedule 📆
+
+There used to be a deployment schedule but now the website is deployed whenever
+a new blog post is approved and merged to the main branch. All other changes are
+deployed as needed.
+
+## How to make changes to the website 👨🏿‍💻
+
+> **Note**
+> Before reading this section, familiarize yourself with [Vercel
+> environments](https://vercel.com/docs/concepts/deployments/environments).
+
+There are primarily two types of website changes, each with its process:
+
+- Content changes ([Storyblok](https://www.storyblok.com/home), our CMS - Content Management System)
+- Code changes (this GitHub repo)
+
+Note that Labs blog posts are an exception. See above:
+[How to publish a new blog post 📝](#how-to-publish-a-new-blog-post-).
+
+This section will cover the process for each type of change.
+
+### Content changes (Storyblok) 📰
+
+Content in Storyblok moves through several stages: drafting -> reviewing -> ready to
+publish -> published -> deployed.
+It's confusing, but "published" in our Storyblok
+configuration does not mean that the content has gone live on the
+corresponding public website. However, you should never publish any content in
+Storyblok if it is not ready for public presentation. In our configuration,
+publishing content in Storyblok is a signal to the rest of the team that says:
+this content is ready at any time to appear on the public website. **If it's not
+ready, don't hit the publish button.**
+
+The Storyblok process is covered in more detail in the [GitHub repo's wiki](https://github.com/Quansight/Quansight-website/wiki).
+A simplified version is presented here.
+
```

Comment:
```suggestion
```

The wiki is now outdated, and has been disabled on the repo. We can remove this mention